### ISSUE TITLE
crit overlay hides chat

### ISSUE THREAD

**Issue by Bod9001** [unitystation/unitystation#6930]
![SPOILER_unknown](https://user-images.githubusercontent.com/68730437/154827349-ff1105c5-fedc-48a8-86af-18ad5fb64952.png)

**Comment by Bod9001** [comment]
humm, technically make sense your unconscious so you can't see what's going on around you but you can't see OOC

**Comment by corp-0** [comment]
Yeah, we’re supposed to interrupt the incoming local messages and replace them with “...” and “you hear something about boots”, not really hiding the whole thing

**Comment by jamessimo** [comment]
Making the Crit a shader will help with this. Have a crit shader on my local and can make a PR later when I have time to test it and configure it to change size on zoom level
